src/Storage.ts

```diff
@@ -88,4 +91,15 @@ export default class Storage {
 			await client.storage.local.set({ [Key.packageVersion]: client.runtime.getManifest().version });
 		}
 	}
+
```

Comment:
In this file I get this error:
```js
index.js:183 
        
       Uncaught (in promise) TypeError: Error in invocation of storage.get(optional [string|array|object] keys, function callback): No matching signature.
    at Function.getIncludeRetweeters (index.js:183:27)
    at I.<anonymous> (index.js:498:39)
    at Generator.next (<anonymous>)
    at index.js:30:32
    at new Promise (<anonymous>)
    at t (index.js:4:10)
    at I.createCheckbox (index.js:491:11)
    at I.<anonymous> (index.js:718:25)
    at Generator.next (<anonymous>)
    at n (index.js:7:10)

```

Reply:
This is fixed by upgrading the manifest version to 3. This API is only supported on V3.

See #90 

https://github.com/dmstern/likers-blocker/pull/90/commits/b4b72168a69261b0b429ff90e82ea50fb79a34dd

Reply:
Great, thank you for your contribution! :)
✔️ Merged.